package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "theia-core",
-  "version": "0.0.1-alpha.2",
+  "version": "0.0.1-alpha.1",
```

Comment:
It's been removed, it was updated automatically after running npm install electron-mocha --save